src/components/common/Calendar/Calendar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import {
+  Wrapper,
+  Event,
+  WeekCalendar,
+  Day,
+  WeekdayTitle
+} from "./StyledCalendar";
+import axios from "axios";
+import { DateTime } from "luxon";
+
+const Calendar = ({ URL, numWeeks }) => {
+  const [eventList, setEventList] = useState([]);
+
+  let componentDidMount = () => {
+    axios
+      .get({ URL })
+      .then(response => {
+        console.log(response);
+        let tempEventList = [];
+        response.data.items.forEach(function(entry) {
+          console.log(entry.summary);
+          let time = DateTime.fromISO(entry.start.dateTime, {
+            zone: entry.start.timeZone
+          }).setZone();
+          tempEventList.push({
+            name: entry.summary,
+            time: time.toFormat("t"),
+            date: time.toLocaleString(),
+            weekday: time.weekday
+          });
+        });
+        console.log(tempEventList);
+        setEventList(tempEventList);
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  };
+
+  React.useEffect(componentDidMount, []);
```

Comment:
React.useEffect(**componentDidMount**, []);
a bit confusing naming, I thought inside functional useEffect you used class based ComponentDidMount() lifecycle method, but here just data is being fetched on mount